website/www/site/content/en/contribute/_index.md

```diff
@@ -58,20 +51,20 @@ detail.
 
 ### Prerequisites
 
-To contribute code, you need
+- a GitHub account
+- a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
+- [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally
+- [Go](https://golang.org) 1.12 or later installed for Go SDK development
+- Python 3.6, 3.7, and 3.8. Yes, you need all four versions installed.
+
+  pip, setuptools, virtualenv, and tox installed for Python development
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
- - Python 3.6, 3.7, and 3.8. Yes, you need all three versions installed.
-    - pip, setuptools, virtualenv, and tox installed for Python development
- - for large contributions, a signed [Individual Contributor License
-   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
-   Software Foundation (ASF).
+- for large contributions, a signed [Individual Contributor License
+  Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+  Software Foundation (ASF).
 
-To install these in a Debian-based distribution:
```

Comment:
I don't think we should drop this.

Reply:
I will change all contents manually at monday and tuesday, and I will try to remember to keep it.